main/main.c

```diff
@@ -60,14 +64,35 @@ static struct argp_option options[] = {
 		"Base directory DIR for Gatekeeper Lua files", 0},
 	{"gatekeeper-config-file", 'f', "FILE", 0,
 		"Lua configuration FILE to initialize Gatekeeper", 0},
+	{"log-file-name-format", 'l', "STRING", 0,
+		"The name format of log files", 0},
+	{"log-base-dir", 'g', "DIR", 0,
+		"Base directory DIR for Gatekeeper log files", 0},
+	{"gatekeeper-log-scan-interval-sec", 'i', "NUM", 0,
+		"Gatekeeper log scan interval in seconds", 0},
 	{ 0 }
 };
 
 struct args {
 	const char *lua_base_dir;
 	const char *gatekeeper_config_file;
+	const char *log_file_name_format;
+	const char *log_base_dir;
+	long gatekeeper_log_scan_interval_sec;
 };
 
+static long
+arg_to_long(const struct argp_state *state, const char *arg)
```

Comment:
Once option `gatekeeper_log_scan_interval_sec` is dropped, we'll no longer need `arg_to_long()`.